editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -56,31 +56,31 @@
 #include "scene/resources/packed_scene.h"
 #include "scene/resources/surface_tool.h"
 
-#define DISTANCE_DEFAULT 4
+constexpr float DISTANCE_DEFAULT = 4;
```

Comment:
All these may need to be `real_t` instead of `float` for consistency with the code that uses them.
Basically if they're used in computations with other `real_t` variables they should likely be `real_t`.